scriptworker/test/test_integration.py

```diff
@@ -5,7 +5,8 @@
 import aiohttp
 import arrow
 import asyncio
-from contextlib import contextmanager
+from asyncio_extras.contextmanager import async_contextmanager
+from async_generator import yield_
```

Comment:
oo TIL (might be worth a filed issue about removing this when min is >= py3.6 due to it "just working" there)